magprime/templates/emails/reg_workflow/attendee_confirmation.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<head></head>
+<body>
+
+You ({{ attendee.full_name }}) have preregistered for {{ c.EVENT_NAME }} this coming {% event_dates %}
+{% if attendee.group %}
+    by claiming one of the badges in the {{ attendee.group.name }} group.
+{% else %}
+    and your payment of ${{ attendee.amount_paid }} has been received.
+{% endif %}
+Your registration confirmation number is {{ attendee.id }}, and you can update your information or kick in
+{% if attendee.amount_extra %}
+    even more
+{% endif %}
+extra money <a href="{{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}">here</a>.
+
+<br/> <br/>
+
+Badges are not mailed out before the event, so your badge will be available for pickup at the registration desk when you arrive at {{ c.EVENT_NAME }}.  Simply bring a photo ID to the registration desk, where you'll be provided with your badge.
+{% if c.DONATIONS_ENABLED %}
+    If you ordered bonus items such as a ribbon, t-shirt, or supporter package, you can pick those up at our merchandise booth.
+    The location and hours of the registration desk and merchandise booth will be e-mailed prior to the event.
+{% endif %}
+
+{% if attendee.amount_extra %}
+    <br/> <br/>
+    Your additional contribution of ${{ attendee.amount_extra }} provides you with these bonus items, which may be picked up at our merch booth:
+    <ul>
+    {% for swag in attendee.donation_swag %}
+        <li>{{ swag }}</li>
+    {% endfor %}
+    </ul>
```

Comment:
Wait, actually.... forget that +1 for a moment.

The way you have this written, it will only mention the extra donation if they ALSO kicked in extra.  But that's probably not what we want, right?